tests/find_iocs_cases/urls.py

```diff
@@ -10,4 +10,36 @@
         {},
         id="URL and domains parsed",
     ),
+    param(
+        "Foo https://citizenlab.ca/about/), bar",
+        {
+            "urls": ["https://citizenlab.ca/about/"],
+        },
+        {"parse_domain_from_url": False},
+        id="URL boundary w/ ) handled properly",
+    ),
```

Comment:
On this failing test, my approach will probably be to remove anything after (and including) a `)` character if there is no `(` in the URL.